src/remote_connector_dao/get_signal.py

```diff
@@ -0,0 +1,29 @@
+import time
+from remote_connector_dao.virtual_channels_builder import (
+    build_coincidences_virtual_channel,
+)
+
+INTEGRATION_TIME = 1e12  # IN PICOSECONDS
+
+
+def get_count_rates(timetagger_proxy, tagger_controller, single_channels_list):
+    tagger_controller.setTriggerLevel(1, 0.09)
+    tagger_controller.setTriggerLevel(2, 0.09)
+
+    coincidences_virtual_channel = build_coincidences_virtual_channel(
+        timetagger_proxy, tagger_controller, single_channels_list
+    )
+
+    channels = [*single_channels_list, coincidences_virtual_channel.getChannels()[0]]
```

Comment:
```suggestion
    coincidences_channels = coincidences_virtual_channel.getChannels()
    channels = [*single_channels_list, coincidences_channels[0]]
```

minor: so para facilitar a leitura

Reply:
ou talvez:

```suggestion
    coincidences_channels = coincidences_virtual_channel.getChannels()
    channels = [*single_channels_list, *coincidences_channels]
```